Source/Blazorise/Components/Badge/Badge.razor

```diff
@@ -6,7 +6,7 @@
         @ChildContent
         @if ( Closable )
         {
-            <span href="#" class="@CloseClassBuilder.Class" aria-label="Close" @onclick="@OnCloseClickedHandler" @onclick:preventDefault tabindex="0" @onkeydown="@OnCloseKeyDownHandler">
```

Comment:
Repeated html structure can be in a fragment if you care to change. Just a suggestion, It can stay as it is eitherway.
![image](https://user-images.githubusercontent.com/22283161/180602623-e333d6ae-2e22-4cac-99a5-88ee65cbe92c.png)

Example : 
![image](https://user-images.githubusercontent.com/22283161/180602660-fa9b03f7-7f57-4ce6-a378-82d59fcac151.png)